### PR TITLE
DOCSP-37310 update loadLevel warning

### PR DESCRIPTION
**Pull request by ltran-mdb2**
## DESCRIPTION

Update loadLevel documentation warning to address bottlenecks

## STAGING

https://preview-mongodbltranmdb2.gatsbyjs.io/cluster-sync/DOCSP-37310/reference/mongosync/#std-option-mongosync.--loadLevel

## JIRA

https://jira.mongodb.org/browse/DOCSP-37310

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=66bf640d8e811ae6726d8cbb

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/includes/opts/loadlevel-warning.rst [hunk 1]
```diff
@@ -1,4 +1,8 @@
 .. warning::
 
    Setting ``loadLevel`` higher than the default of ``3`` may negatively
-   impact the destination cluster performance.
\ No newline at end of file
+   impact the destination cluster performance.
+
+   If read bottlenecks exist on the source cluster or if write
+   bottlenecks exist on the destination cluster, decreasing
+   ``loadLevel`` may improve performance.
```
Comment (jmd-mongo) [inline source/includes/opts/loadlevel-warning.rst:8]: [suggestion]

Consider updating to "might improve performance". 

Source: https://www.mongodb.com/docs/meta/style-guide/writing/use-effective-verbs/#use-modal-verbs-accurately

**Review by jmd-mongo — COMMENTED** [review]
Thanks for this helpful addition, @ltran-mdb2! I've left a suggestion for your consideration.

Thanks,
Joe

File: source/includes/opts/loadlevel-warning.rst (hunk 1 above)
Comment (ltran-mdb2) [reply source/includes/opts/loadlevel-warning.rst]: Done!

**Review by jmd-mongo — APPROVED** [review]
Thanks for this update, @ltran-mdb2! This one LGTM.

Thanks,
Joe

**Review by suspicious-mango — APPROVED** [review]
LGTM

**Pull request merged**